plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduMetadata.java

```diff
@@ -171,7 +171,7 @@ private ConnectorTableMetadata getTableMetadata(KuduTableHandle tableHandle)
                 .collect(toImmutableList());
 
         Map<String, Object> properties = clientSession.getTableProperties(tableHandle);
-        return new ConnectorTableMetadata(tableHandle.getSchemaTableName(), columnsMetaList, properties);
+        return new ConnectorTableMetadata(tableHandle.getSchemaTableName(), columnsMetaList, properties, Optional.ofNullable(table.getComment()).filter(s -> !s.isEmpty()));
```

Comment:
Please remove `filter()`.

Reply:
How should I make sure that an empty string is passed as a `null` value ?